runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -2816,3 +2829,11 @@ JITaaSHelpers::getRemoteROMClass(J9Class *clazz, JITaaS::J9ServerStream *stream,
    *classInfoTuple = std::get<0>(recv);
    return romClassFromString(std::get<0>(*classInfoTuple), trMemory->trPersistentMemory());
    }
+
+void
+JITaaSHelpers::getRemoteRomClassReturnClassinfo(J9Class * clazz, JITaaS::J9ServerStream *stream, ClientSessionData::ClassInfo &classInfo)
+   {
+   JITaaSHelpers::ClassInfoTuple classInfoTuple;
+   auto romClass = JITaaSHelpers::getRemoteROMClass(clazz, stream, TR::comp()->trMemory(), &classInfoTuple);
```

Comment:
I would like to minimize the use of thread locals if at all possible because they are expensive operations. trMemory and clientData could be available at the caller of this method.